task_sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -204,7 +219,7 @@ def xcom_pull(
         key: str = "return_value",  # TODO: Make this a constant (``XCOM_RETURN_KEY``)
         include_prior_dates: bool = False,  # TODO: Add support for this
         *,
-        map_indexes: int | Iterable[int] | None = None,
+        map_index: int | None | ArgNotSet = NOTSET,
```

Comment:
Does this impact users?

Reply:
Yup, totally. Removing it.

Nice catch